src/averageAges.js

```diff
@@ -55,7 +74,16 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = people.filter(child => onlyWithSon
+    ? people.find(person => child.mother === person.name && child.sex === 'm')
+    : people.find(person => child.mother === person.name));
+
+  const sumArray = children
```

Comment:
naming
if some comment was left by mentor, make sure you fixed it everywhere in your code

Reply:
sumArray is a bad example, it's clear it's an array, better explain the sum of what you have
```suggestion
  const sumOfAges = children
```